src/commands/admin/jobs/start.js

```diff
@@ -24,6 +30,30 @@ module.exports = ({
     if (!job.meta.guildControlled) {
       return await message.lineReply('Unable to enable a job that is not guild controlled.')
     }
+
+    // user provided interval
+    if (intervalText) {
+      if (!validateText(intervalText, INTERVAL_FORMAT)) {
+        return await message.lineReply(`Specified format is incorrect. The correct format is ${secsToDHMS(0)}.`)
+      }
+
+      const timeValidationMsg = validateTime(intervalText)
+      if (timeValidationMsg !== '') {
+        return await message.lineReply(timeValidationMsg)
+      }
+
+      interval = parseInterval(intervalText)
+      if (job.meta.defaultInterval >= interval) {
+        return await message.lineReply(`The interval cannot be lesser or equal to ${secsToDHMS(Math.floor(job.meta.defaultInterval / 1000))}.`)
+      }
+
+      interval = Math.floor(interval / 1000)
+      lastJob = getISODate()
+
+    } else if (job.meta.defaultInterval) {
+      interval = Math.floor(job.meta.defaultInterval / 1000)
+      lastJob = getISODate()
+    }
```

Comment:
Should we have an `else` here for catching an error if `intervalText` is null / empty?

Reply:
I don't think so, if intervalText is empty, then it will just use defaultInterval, if there is no defaultInterval, then the job will be run on the interval that runs the code.